.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 9 # Current stable 
+- 9 # Current stable
```

Comment:
Want to skip this in the pr?

Reply:
What do you mean? It's my editor which removed a trailing space on save, it's a pain to exclude every time...

Reply:
You just skip the file when you rebase, as there are no other changes in the file.

Yeah my editor does that too, but this file is untouched in this PR.

Reply:
Done, thanks for pointing this out.